python/common/org/python/stdlib/datetime/DateTime.java

```diff
@@ -60,36 +83,42 @@ public DateTime(org.python.Object[] args, java.util.Map<java.lang.String, org.py
                 this.timeUnits[keyIndex] = ((org.python.types.Int) args[argIndex]).value;
```

Comment:
The exception above is never tested, in case you want full code coverage you might want to test it

Reply:
Added a test in 946bb34, is covered now.